src/util/git/git-spawn.js

```diff
@@ -0,0 +1,15 @@
+/* @flow */
+
+import * as child from '../child.js';
+
+// Suppress any password prompts since we run these in the background
+const env = {
+  GIT_ASKPASS: '',
+  GIT_TERMINAL_PROMPT: 0,
+  GIT_SSH_COMMAND: 'ssh -oBatchMode=yes',
+  ...process.env,
+};
+
+export const spawn = (args: Array<string>, opts?: child_process$spawnOpts = {}): Promise<string> => {
```

Comment:
That mean I would mock the spawn function and test the spawn function in the same test file, I think I will end up testing the mock and not the actual code.
I will delay this question after another PR that I will prepare about git hosted resolvers.